server/tests/host_test.cpp

```diff
@@ -102,7 +102,7 @@ int main(int argc, char* argv[])
         delete encrypted_new_params_ptr[i];
     }
 
-    map<string, vector<double>> new_params = deserialize(serialized_new_params);
+    map<string, vector<float>> new_params = deserialize(serialized_new_params);
```

Comment:
This file was not updated to use floats on master branch

Reply:
Oh you're right, my mistake